src/validateFunctions.js

```diff
@@ -0,0 +1,9 @@
+export const createOutput = (error, parsedData, rawData) => {
+    const response =  {
+        valid: error.length > 0,
+        error,
```

Comment:
This should be in plural form. This should be an array of objects that contains the errors that were encountered during the validation.
The format of the errors should follow the definition in #6 
Extract:
```
{
      "type": <missing-header|parameter-value|parameter-encoding|missing-parameter|invalid-header|unknown-parameter|invalid-header-encoding|invalid-json|etc...>,
      "key": <string>
      "value": <any-type>
      "description": <string (description for humans)>
}
```